### PR TITLE
Try quoting labels in release notes config

### PR DESCRIPTION
**Pull request by armanbilge**
Not sure if this will fix it, but something seems to be broken 😕 

```
Could not parse .github/release.yml: Tried to load unspecified class: Symbol
```

### CODE REVIEW

**Comment by armanbilge** [comment]
Nice, that fixed it :)

File: .github/release.yml [hunk 1]
```diff
@@ -2,14 +2,14 @@ changelog:
   categories:
     - title: Features
       labels:
-        - :mushroom: enhancement
+        - ':mushroom: enhancement'
```
Comment (djspiewak) [inline .github/release.yml:5]: Oh yeah, the problem here is the `:`. Quoting is the correct fix.

Comment (armanbilge) [reply .github/release.yml:5]: Other YAML parsers like it just fine. /shrug

Comment (djspiewak) [reply .github/release.yml:5]: I'm pretty sure that's out of spec, but I'd have to check. My recollection is that any unquoted string which contains a non-leading `:` should be considered a key/value pair.

**Pull request merged**